services/src/Microsoft.Azure.IIoT.Services.OpcUa.Publisher/src/Startup.cs

```diff
@@ -223,8 +223,6 @@ public class Startup {
                 .AsImplementedInterfaces().SingleInstance();
             builder.RegisterType<CosmosDbServiceClient>()
                 .AsImplementedInterfaces();
-            builder.RegisterType<DefaultJobService>()
```

Comment:
Sure?

Reply:
It is a duplicate registration. Another registration can be found on line `218`.